packages/gestalt/src/MasonryV2.js

```diff
@@ -322,12 +322,13 @@ function useLayout<T: { +[string]: mixed }>({
   positions: $ReadOnlyArray<?Position>,
   updateMeasurement: (T, number) => void,
 } {
-  const hasMultiColumnItems =
-    _twoColItems &&
-    items
-      .filter((item) => item && !positionStore.has(item))
-      .some((item) => typeof item.columnSpan === 'number' && item.columnSpan > 1);
-  const itemToMeasureCount = hasMultiColumnItems ? MULTI_COL_ITEMS_MEASURE_BATCH_SIZE : minCols;
+  const multiColumnItemsCount = _twoColItems
```

Comment:
hmmm this doesn't seem right- in the case where we measure all items and start positioning them, there's actually nothing that would trigger a re-render after the first "batch" has been positioned

is there a reason the layout function still couldn't just return the positions for all items? e.g. split the graph into multiple batches but still return the full set?